draft-ietf-tls-subcerts.md

```diff
@@ -327,15 +332,16 @@ This document defines the following extension code point.
    } ExtensionType;
 ~~~~~~~~~~
 
+### Server authentication
+
 A client which supports this specification SHALL send an empty
-"delegated_credential" extension in its ClientHello.  If the client receives a
-delegated credential without indicating support, then the client MUST abort with
-an "unexpected_message" alert.
+"delegated_credential" extension in its ClientHello.  If the server receives a
```

Comment:
Since this section is about server authentication, shouldn't we specify what the client must do if it receives an unsolicited DC? (That is, I think the previous text using `client` was correct.)

Reply:
Good catch, will fix.